sharepa/analysis.py

```diff
@@ -10,17 +9,50 @@ def bucket_to_dataframe(name, bucket):
         :type bucket: list[dict]
         :returns: pandas.DataFrame
     '''
-    return pd.DataFrame(
-        {item['key']: item['doc_count'] for item in bucket},
-        index=[name]
-    ).T
+    list_of_dicts = []
```

Comment:
it would be better to give this a name that reflects what it is doing/storing. It is a list of dicts, but the actual purpose is a little more specific than that, and tying variable names to something with some meaning can help make code more readable.